setup/quickstart/halyard-gke-deploy-rbac/index.md

```diff
@@ -0,0 +1,159 @@
+In many cases you want to deploy to clusters in another GCP project that also have Kubernetes RBAC enabled.
+
+In this codelab, we will enable Spinnaker to push your containers to another GCP project that have a cluster secured by RBAC. This codelab will allow Spinnaker to be able to manipulate all deployments in the target cluster. 
+You may want to limit Spinnaker further but how to do that is not covered by this tutorial. 
+This guide won't show you how to setup Spinnaker in a RBAC cluster.
+
+## Overview
+
+At the end of this guide you will have
+
+* A new Kubernetes provider with RBAC enabled
+* Ability to deploy to a cluster in another GCP project
+
+## Part 0: Prerequisites
+
+A GCP project (referred to as $GCP_PROJECT) running a Spinnaker [GKE cluster with halyard](/setup/quickstart/halyard-gke/)
+
+Another K8s cluster (referred to as $K8_TEST) running with Kubernetes version > 1.6 on another GCP project ($GCP_TEST)
+
+## Part 1: Configure Gcloud
+
+Make sure that you are authenticated against the test cluster ($K8_TEST). 
+
+```bash
+gcloud info
+```
+
+If you are unfamiliar with how to use multiple project with gcloud you can use this [guide](https://cloud.google.com/sdk/docs/managing-configurations) to get started
+
+Set your zone and project variables and get the credentials for the kubernetes cluster
+
+```bash
+GCP_PROJECT=my-spinnaker-project
+ZONE=us-central1-f
+K8_TEST=my-test-cluster
+GCP_TEST=my-test-project
+gcloud container clusters get-credentials $K8_TEST --zone $ZONE --project $GCP_TEST
+```
+
+## Part 2: Add Service account to GCP
+
+Create the service account in GCP test project (note that you get the service account from your Spinnaker project)
+
+````bash
+HALYARD_SA=halyard-service-account
+
+#Get the SA email from the Spinnaker project
+HALYARD_SA_EMAIL=$(gcloud iam service-accounts list \
+    --project=$GCP_PROJECT \
+    --filter="displayName:$HALYARD_SA" \
+    --format='value(email)')
+
+#Add policy bindings in the Test project
+gcloud projects add-iam-policy-binding $GCP_TEST \
+    --role roles/iam.serviceAccountKeyAdmin \
+    --member serviceAccount:$HALYARD_SA_EMAIL
+
+gcloud projects add-iam-policy-binding $GCP_TEST \
+    --role roles/container.admin \
+    --member serviceAccount:$HALYARD_SA_EMAIL
+```` 
+## Part 3: Add a service account (SA) to kubernetes
+
+Next we need to add a service account to kubernetes that will handle the authorization inside the kubernetes cluster.
+Create a file, spinnaker-service-account.yaml, with the following content:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: spinnaker-service-account
+  namespace: default
+```
+Apply the service account
+
+```bash
+kubectl apply -f ./spinnaker-service-account.yaml
+```
+
+Then we need to set access for Spinnaker to edit the cluster in order to let Spinnaker manage deployments to the cluster
```

Comment:
"Then we need  to set access for Spinnaker to edit the cluster so it can manage deployments there."